### ISSUE TITLE
test case failure

### ISSUE THREAD

**Issue by pnemade** [andris9/gettext-parser#13]
Hi,
  I am packaging gettext-parser module for Fedora and running the tests as
 ````
$ mocha -R spec test/*


  Folding tests
    ✓ Short line, no folding 
    ✓ Short line, force fold with newline 
    ✓ Long line 

  MO Compiler
    UTF-8
      1) should compile
    Latin-13
      2) should compile

  MO Parser
    UTF-8
      ✓ should parse 
    Latin-13
      ✓ should parse 

  PO Compiler
    UTF-8
      ✓ should compile 
    Latin-13
      ✓ should compile 

  PO Parser
    UTF-8
      ✓ should parse 
    UTF-8 as a string
      ✓ should parse 
    Stream input
      ✓ should parse (79ms)
    Latin-13
      ✓ should parse 


  11 passing (231ms)
  2 failing

  1) MO Compiler UTF-8 should compile:

      AssertionError: expected { Object (0, 1, ...) } to deeply equal { Object (0, 1, ...) }
      + expected - actual

         "20": 0,
         "21": 0,
         "22": 0,
         "23": 0,
      +  "24": 76,
      -  "24": 124,
         "25": 0,
         "26": 0,
         "27": 0,
         "28": 0,
         "688": 197,
         "689": 161,
         "690": 0,
         "length": 691,
      +  "offset": 5504,
      -  "offset": 6808,
         "parent": {
           "0": 39,
           "1": 117,
           "2": 115,
      
      at Assertion.assertEql (/usr/lib/node_modules/chai/lib/chai/core/assertions.js:489:10)
      at Assertion.ctx.(anonymous function) [as eql] (/usr/lib/node_modules/chai/lib/chai/utils/addMethod.js:40:25)
      at Assertion.assertEqual (/usr/lib/node_modules/chai/lib/chai/core/assertions.js:455:19)
      at Assertion.ctx.(anonymous function) [as equal] (/usr/lib/node_modules/chai/lib/chai/utils/addMethod.js:40:25)
      at Context.<anonymous> (/home/parag/rpmbuild/BUILD/gettext-parser-1.1.1/test/mo-compiler-test.js:18:38)
      at callFn (/usr/lib/node_modules/mocha/lib/runnable.js:223:21)
      at Test.Runnable.run (/usr/lib/node_modules/mocha/lib/runnable.js:216:7)
      at Runner.runTest (/usr/lib/node_modules/mocha/lib/runner.js:374:10)
      at /usr/lib/node_modules/mocha/lib/runner.js:452:12
      at next (/usr/lib/node_modules/mocha/lib/runner.js:299:14)
      at /usr/lib/node_modules/mocha/lib/runner.js:309:7
      at next (/usr/lib/node_modules/mocha/lib/runner.js:247:23)
      at Object._onImmediate (/usr/lib/node_modules/mocha/lib/runner.js:276:5)
      at processImmediate [as _immediateCallback] (timers.js:354:15)

  2) MO Compiler Latin-13 should compile:

      AssertionError: expected { Object (0, 1, ...) } to deeply equal { Object (0, 1, ...) }
      + expected - actual

         "20": 0,
         "21": 0,
         "22": 0,
         "23": 0,
      +  "24": 76,
      -  "24": 124,
         "25": 0,
         "26": 0,
         "27": 0,
         "28": 0,
         "694": 254,
         "695": 240,
         "696": 0,
         "length": 697,
      +  "offset": 2408,
      -  "offset": 4544,
         "parent": {
           "0": 123,
           "1": 10,
           "2": 32,
      
      at Assertion.assertEql (/usr/lib/node_modules/chai/lib/chai/core/assertions.js:489:10)
      at Assertion.ctx.(anonymous function) [as eql] (/usr/lib/node_modules/chai/lib/chai/utils/addMethod.js:40:25)
      at Assertion.assertEqual (/usr/lib/node_modules/chai/lib/chai/core/assertions.js:455:19)
      at Assertion.ctx.(anonymous function) [as equal] (/usr/lib/node_modules/chai/lib/chai/utils/addMethod.js:40:25)
      at Context.<anonymous> (/home/parag/rpmbuild/BUILD/gettext-parser-1.1.1/test/mo-compiler-test.js:27:38)
      at callFn (/usr/lib/node_modules/mocha/lib/runnable.js:223:21)
      at Test.Runnable.run (/usr/lib/node_modules/mocha/lib/runnable.js:216:7)
      at Runner.runTest (/usr/lib/node_modules/mocha/lib/runner.js:374:10)
      at /usr/lib/node_modules/mocha/lib/runner.js:452:12
      at next (/usr/lib/node_modules/mocha/lib/runner.js:299:14)
      at /usr/lib/node_modules/mocha/lib/runner.js:309:7
      at next (/usr/lib/node_modules/mocha/lib/runner.js:247:23)
      at Object._onImmediate (/usr/lib/node_modules/mocha/lib/runner.js:276:5)
      at processImmediate [as _immediateCallback] (timers.js:354:15)



error: Bad exit status from /var/tmp/rpm-tmp.4nq5EC (%check)

````

Any idea how to fix this?

**Comment by pnemade** [comment]
Just saw the recent commit fixed above issue. Can it be possible to tag new version here so that I can pick that tagged commit and build in Fedora?

**Comment by andris9** [comment]
Published as v1.1.2

**Comment by pnemade** [comment]
Thank you.
